NuRadioMC/EvtGen/NuRadioProposal.py

```diff
@@ -325,6 +320,12 @@ def __get_propagator(self, particle_code=13):
                     "path {} exists.".format(config_file_full_path))
 
             pp.InterpolationSettings.tables_path = self.__tables_path
+            # download pre-calculated tables for default configs, but not more than once
+            if self.__config_file in config_files and not self.__tables_downloaded:
+                from NuRadioMC.EvtGen.proposal_table_manager import download_proposal_tables
+
+                download_proposal_tables(self.__config_file, tables_path=self.__tables_path)
+                self.__tables_downloaded = True
```

Comment:
If I understand correctly, this means e.g. every time a new Python script is launched, the tables will be redownloaded? Is that intentional?

Reply:
hmm... yes maybe not too wise... especially when launching thousands of jobs.

I wanted to avoid that a downloader script must be run manually. 
Do you have a good suggestion to check if latest set of tables exists already (should only change if we swap to different proposal version at some point)?

 ... without thinking in detail, one could eg. create a subdirectory structure `<proposal_tables>/Greenland` (or a more detailed one not omitting the compiler and proposal version used) or dumping the already downloaded paths to a `<proposal_tables>`/downloaded_tables.csv after successful download and checking that?

Reply:
My first idea was also along the lines of your first suggestion (although I would include the proposal version, at least, to avoid unexpected behaviour in the future if something should change), though I guess keeping a record of downloaded proposal tables (probably with hashes?) also works. 
The former means things get a bit trickier if the user wants to modify the default download path, the latter would lead to errors if the user manually (re)moves tables. I think I would be fine with either.

Reply:
Nothing would lead to errors. Worst case tables are re-computed if not found / if table hashes changed.

The list with hashes you proposed to check for equivalence is a bit difficult because we are downloading and extracting a tarball. But then I'll go with the first and put default tables to a sub-directory structure `<tables_path>/<config_file>`

Reply:
With the implemented sub-directory structure the `<config_file>` part in the `self.__tables_path` now set in the init can easily be adapted should version/etc for the download ever become an issue in the future.